parm/config/config.aeroanl

```diff
@@ -5,13 +5,19 @@
 
 echo "BEGIN: config.aeroanl"
 
-export OBS_YAML_DIR=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/config/
-export OBS_LIST=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/lists/aero_prototype.yaml
-export AEROVARYAML=$HOMEgfs/sorc/gdas.cd/parm/aero/variational/3dvar_dripcg.yaml
-export BERROR_YAML=$HOMEgfs/sorc/gdas.cd/parm/aero/berror/static_bump.yaml
-export FV3JEDI_FIX=$HOMEgfs/fix/gdas
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/lists/gdas_aero_prototype.yaml
+export AEROVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/variational/3dvar_gfs_aero.yaml
+export BERROR_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/aero/berror/
+export BERROR_YAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/berror/staticb_bump_aero.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+export BERROR_TYPE='identity'
```

Comment:
g-w PR #1104 added variable `STATICB_TYPE` to identify the type (source) of staticb to use.  Valid values are `bump`, `gsibec`, and `identity`.   Is `BERROR_TYPE` focused more generally on the type of background error covariance (ie, hybrid or pure variational) or the type of fixed (static) background error covariance to use?    Looking at `feature/aeroda_staticb_crm` it seems it's the later.   If true, should we replace `BERROR_TYPE` with `STATICB_TYPE` to be more explicit?   Alternatively, we replace `STATICB_TYPE` with `BERROR_TYPE`.  Either way, it looks like one variable will suffice for aerosol and atmospheric UFS DA.

Reply:
I don't have a strong preference on which convention is used, but I agree it should be consistent. I will update mine to use `STATICB_TYPE`

Reply:
Thanks